README.md

```diff
@@ -1,38 +1,131 @@
+<br />
+
+<h3 align="center">NakVaksin</h3>
+
+<p align="center">
+A simple, easier way to get notified on your appointment.
+<br />
+<br />
+<a href="https://nakvaksin.vercel.app/">View The App</a>
```

Comment:
Can change to "View the App (Under development)"? Apparently we are getting traffic, and I dont wish to give the wrong idea since the app is not completed yet

Reply:
We might also want to add `Under development` in our home page, lets think of something